### PR TITLE
Updated jQuery, added new Backgrounds, added Menu Theme

### PR DESCRIPTION
**Pull request by nnsdev**
Updated jQuery to newest version.
Replaced some Backgrounds showing the full rendering power of Terasology.
Added Menu Theme from Chrisk to Page.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -158,7 +162,7 @@
 				</td>
 			</tr>
 			<div id="footer">
-			<p>&copy; 2013 Terasology Contributors - credits: Mahmood Dhia, NowNewStart, MiJyn</p>
+			<p>&copy; MovingBlocks 2013 Created by: 3000Lane, NowNewStart, MiJyn</p>
```
Comment (AnonymousMeerkat) [inline index.html]: Would "Terasology website" be better than "MovingBlocks"? Idk, it just seems vague to me

**Comment by nnsdev** [comment]
"MovingBlocks","Terasology" or "Terasology Website"?
EDIT:
What's about "MovingBlocks & it's Contributors" as replacement to "MovingBlocks - Contributors:..." and a link on Contributors displaying a list of them?

**Comment by Cervator** [comment]
Haven't forgotten, but it has gotten too late - will check tomorrow :-)

**Comment by nnsdev** [comment]
I think it's the best Solution we write "MovingBlocks 2013 Website created by:".

**Comment by Cervator** [comment]
Okay, started looking and I thought something was off - your fork is three months out of date and missing a bunch of stuff. It replaces backgrounds that have already been replaced, etc.

I suggest saving your changes somewhere safe then refresh your fork and redo the needed changes. Closing this in the meantime. Will try to figure out the footer thing this weekend in the license forum thread, I had a todo item myself to update it anyway :-)

**Comment by nnsdev** [comment]
Lol oops :D Yeah will do when I have time for it :)

Date: Sat, 13 Jul 2013 13:21:36 -0700

**Pull request closed without merging**